code/go/internal/validator/folder_item_spec.go

```diff
@@ -89,7 +100,17 @@ func (s *folderItemSpec) validate(fs http.FileSystem, folderSpecPath string, ite
 
 	var errs ValidationErrors
 	for _, re := range result.Errors() {
-		errs = append(errs, fmt.Errorf("field %s: %s", re.Field(), re.Description()))
+		errs = append(errs, fmt.Errorf("field %s: %s", re.Field(), adjustErrorDescription(re.Description())))
 	}
 	return errs
 }
+
+func loadRelativePathFormatChecker(currentPath string) {
+	gojsonschema.FormatCheckers.Add("relative-path", RelativePathChecker{
```

Comment:
Consider making `relative-path` a constant, now that we are using it in a couple of places in code.

Reply:
Fixed